test/e2e/apimachinery/garbage_collector.go

```diff
@@ -452,7 +465,13 @@ var _ = SIGDescribe("Garbage collector", func() {
 		gatherMetrics(f)
 	})
 
-	It("should orphan pods created by rc if deleteOptions.OrphanDependents is nil", func() {
+	/*
+		    Testname: garbage-collector-delete-rc-orphaning-unset
+		    Description: Ensure that if deleteOptions.OrphanDependents is set to nil,
```

Comment:
This just requires a change in the comment. Both `OrphanDependents` and `PropagationPolicy` are default to nil.